README.md

```diff
@@ -10,39 +10,45 @@ All requirements are already included in macOS. Patches are welcome for portabil
 
 ## Usage
 
-```
+```shell
 download-windows-esd languages
 ```
 
+  - Or use a one-liner
+
+    ```shell
+    curl -fsSL https://raw.githubusercontent.com/mattieb/download-windows-esd/refs/heads/main/download-windows-esd | bash -s -- download en-us Core ARM64
+    ```
```

Comment:
Few things here: 

- This isn't how to use the "languages" subcommand, which this example and text is talking about.
- Stylistically, let's not do a single bullet point like this.
- I'm not completely opposed to talking about the "curl | sh" pattern as an option, since this could often be a tool people use once and never again, but I think it's better explained in a "getting the tool" section preceding "usage".

Reply:
> could you please offer a [pull-request suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) (refer to no. 7) to save the back-and-forth?

I can't use the suggest feature to make the change I need, which would remove this change entirely from this area and instead propose that you could move talking about the "curl | sh" pattern into a brand-new section of the README talking about how to download or run the tool as a one-off.

Likely the easiest thing at this point would be for you to roll back most README changes and I can just make that change separately at a later time.